### DISCUSSION TITLE
Link 404

### DISCUSSION THREAD

**Discussion by tristan-garaud** [vercel/next.js#52451] [Help]
### Summary

Hi,
I'm trying to do something really basic.
Trying to create a route by adding a component about inside the pages folder and then go to localhost:300/about and I just have 404. I was thinking it was maybe because client side component so I've put "use client" at some places but still the same problem. He is the code:

```
export function HomeMain() {
    return (<div>
        <div className="flex justify-center">
            <p className="inline-block text-[40px] text-[red]">Veterinario</p>
        </div>
        <div className="flex">
            <div className="inline-block shrink-0 pr-[40px] w-[200px]">
                <p className="text-[24px] hover:text-[28px] text-[black] cursor-pointer">Profile</p>
                <Link href="/animal/animals">
                    <p className="text-[24px] hover:text-[28px] text-[black] cursor-pointer">Patients</p>
                </Link>
                <p className="text-[24px] hover:text-[28px] text-[black] cursor-pointer">Schedule</p>
                <p className="text-[24px] hover:text-[28px] text-[black] cursor-pointer">Consultation</p>
                <Link href="/about">
                    <p className="text-[24px] hover:text-[28px] text-[black] cursor-pointer">About</p>
                </Link>
            </div>
        </div>
    </div>)
}
```
```
export default function About() {
    return (<div>
        About
    </div>)
}
```

**Comment by icyJoseph** [comment]
Could you share your folder structure?

If you are using pages directory, having `pages/about.tsx` should be enough, if you are using appDir, you need `app/about/page.tsx`

**Reply by tristan-garaud** [reply]
Yes I fixed it by moving the pages folder at the root of the project. before it was in src/app/.
There is no way to define where the pages folder should be ?

**Reply by icyJoseph** [reply]
Not really, both pages and app must be at the root of the project, or `src` if you use the src folder, to be picked up as routes.
